src/ape/plugins/__init__.py

```diff
@@ -120,7 +120,11 @@ def __init__(self):
         for _, name, ispkg in pkgutil.iter_modules():
             if name.startswith("ape_") and ispkg:
                 try:
-                    plugin_manager.register(importlib.import_module(name))
+                    module = importlib.import_module(name)
+                    if plugin_manager.is_registered(module):
+                        plugin_manager.unregister(module)
+
+                    plugin_manager.register(module)
```

Comment:
Why not this? I don't get why unregistering the module here
```suggestion
                    if not plugin_manager.is_registered(module):
                        plugin_manager.register(module)
```

Reply:
If you switch projects in the same python session, the plugin configs don't reload otherwise.
This was causing issues in the tests.

Reply:
Like it will still have the plugin configs from the first original project. I am not sure the best way to handle this. It has been really difficult to try and test this.

Reply:
Hmm, yes this would seem to be a big issue then

Reply:
Wait, why are you trying to load configs from another project?

Reply:
This was no longer needed thank to your suggestions for trying to load these things more lazily. So far, it seems like this is a success!